<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_device_laptop</class>
 <widget class="QDialog" name="edit_device_laptop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="OK_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background: rgb(45, 81, 135);
border-radius: 12px;</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>621</width>
     <height>306</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="os_lineEdit"/>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="weight"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Battery:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>OS:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Weight:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="gaming">
      <property name="text">
       <string>Gaming</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="motherboard"/>
    </item>
    <item row="7" column="3">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>Watt</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Memory:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="memory"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="battery_watt">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="dimension2_lineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Motherboard:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QLineEdit" name="dimension4_lineEdit_2"/>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLineEdit" name="dimension3_lineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Dimension:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="touchscreen">
      <property name="text">
       <string>Touch screen</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="dimension1_lineEdit"/>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="battery_cell">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Cell</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>201</width>
     <height>302</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="model_lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Brand:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="brand_lineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Price:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="price_lineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Quantity:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="quantity_lineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>CPU:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="cpu"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>GPU:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="gpu"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>Display:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="display">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="des_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
color: rgb(0, 85, 127);</string>
   </property>
   <property name="text">
    <string>EDIT (LAPTOP)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
